node/sha3pow/Cargo.toml

```diff
@@ -9,11 +9,11 @@ parity-scale-codec = '3.1.2'
 sha3 = "0.10.1"
 rand = { version = "0.8.5", features = ["small_rng"] }
 
-sc-consensus = {git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.42" }
-sc-consensus-pow = {git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.42" }
-sc-client-api = {git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.42" }
-sp-api = {git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.42" }
-sp-blockchain = {git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.42" }
-sp-consensus-pow = {git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.42" }
-sp-core = {git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.42" }
-sp-runtime = {git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.42" }
+sc-consensus = {git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.43" }
+sc-consensus-pow = {git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.43" }
+sc-client-api = {git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.43" }
```

Comment:
same here, inconsistent brackets formatting

Reply:
Oh yeah, it's on a bunch of them. I never noticed.